app/controllers/v0/education_benefits_claims_controller.rb

```diff
@@ -0,0 +1,23 @@
+module V0
```

Comment:
What @austin-martinez said, plus `Vx::MyController` is what I've seen most. You should also have the latest version available without the version # in the URL. This smells a bit like premature optimization, the pattern we followed was to not have a Vx module until it's needed. Once you do need it you move the old code down into a module and the latest remains up top. Routing handles the rest (no pun intended) of the modules that haven't changed.